PowerEditor/src/NppCommands.cpp

```diff
@@ -3745,8 +3745,8 @@ void Notepad_plus::command(int id)
 		case IDM_VIEW_CURLINE_HILITING:
 		{
 			COLORREF colour = (NppParameters::getInstance()).getCurLineHilitingColour();
-			_mainEditView.setCurrentLineHiLiting(!_pEditView->isCurrentLineHiLiting(), colour);
-			_subEditView.setCurrentLineHiLiting(!_pEditView->isCurrentLineHiLiting(), colour);
+			_mainEditView.setCurrentLineHiLiting(_pEditView->isCurrentLineHiLiting(), colour);
+			_subEditView.setCurrentLineHiLiting(_pEditView->isCurrentLineHiLiting(), colour);

```

Comment:
Use `NppParameters::getInstance().getSVP()._currentLineHilitingShow;`  instead of `isCurrentLineHiLiting`.

Reply:
Done. Using a bool variable to do it once instead of two separate lookups.